kili/utils/pagination.py

```diff
@@ -148,13 +166,9 @@ def generate_variables(batched_properties):
     """
     results = []
     for batch_number, batch in enumerate(batch_object_builder(properties_to_batch, batch_size)):
-        mutation_start = time.time()
-        variables = generate_variables(batch)
-        result = self.auth.client.execute(request, variables)
-        mutation_time = time.time() - mutation_start
+        payload = generate_variables(batch)
+        result = api_throttle(self.auth.client.execute)(request, payload)
```

Comment:
not necessary to do right now, but at some point we should disable throttling for signed url upload

Reply:
but the backend will still have the limitation when calling the append_many_to_dataset_resolver isn't it ?

Reply:
yes, forget what I said